verif/cache/ref_orderer.py

```diff
@@ -0,0 +1,56 @@
+import os
+import itertools
+
+
+
+def orderer_func(arg1):
+    N = 6
+    input_filename = arg1
+    output_filename = "verif/cache/output.log"
+    temp_filename  = "verif/cache/temp.log"
+
+    # Open input and output files
+    with open(input_filename, 'r') as input_file, open(output_filename, 'w') as output_file:      
+            
+        lines = input_file.readlines()
+        #Write the header
+        for line in lines:
+            if N != 0:
+                N = N-1
+                output_file.write(line)
+
+        while (lines):
+            with open(temp_filename, 'w') as temp_file:
+                # Find the first occurrence of "CORE_RD_REQ"
+                input_file.seek(0) 
+                lines = input_file.readlines()
+                core_rd_req_found = False
+                cache_rd_rsp_found = False
+                for line in lines:
```

Comment:
I'm no python expert, but i think a "while loop" would make more sense here.
So you can "break" the look one you find the first "CORE_RD_REQ".